src/Http/Controllers/InstallController.php

```diff
@@ -60,38 +61,57 @@ public function index(): Response
             }
         }
 
-        Craft::$app->getView()->registerAssetBundle(InstallerAsset::class);
-
         // Grab the license text
         $licensePath = Aliases::get('@craftcms/LICENSE.md');
         $license = file_get_contents($licensePath);
+        $licenseHtml = Str::markdown($license);
 
         // Guess the site name based on the server name
         $defaultSystemName = $this->defaultSiteName();
         $defaultSiteUrl = $this->defaultSiteUrl();
         $defaultSiteLanguage = $this->defaultSiteLanguage();
+        $locales = Craft::$app->getI18n()->getAllLocales();
+        $connection = config('database.default');
+        $dbConfig = config("database.connections.{$connection}");
```

Comment:
```suggestion
        $dbConfig = DB::getConfig();
```

I think this results in the same data